src/ml_tooling/baseclass.py

```diff
@@ -28,6 +30,11 @@
 
 logger = create_logger("ml_tooling")
 
+from enum import Enum
+class Environment(Enum):
```

Comment:
Is this not used for anything yet?

Reply:
no, not yet. the idea was to use it to distinguish the environments one is using to storage for. but it doesnt make much sense for the filestorage as it will mostly by used for development.
maybe it could be used for other storage classes?

Reply:
It is more that it is located in baseclass file where it is unused. It makes sense to have it in the storage file where there already is the StorageEnvironment

Reply:
ah yes indeed, this is something i forgot to delete :D thanks!